redux/cartSlice.ts

```diff
@@ -41,9 +38,19 @@ const cartSlice = createSlice({
       );
       state.cart = removeItem;
     },
+    clearCart: (state, action) => {
+      while (state.cart.length > 0) {
+        state.cart.pop();
+      }
+    },
```

Comment:
You can also reset the cart by doing: `state.cart = []`

Reply:
Of course! Will do. Thanks!